src/shared/cgroup-setup.c

```diff
@@ -154,6 +154,21 @@ bool cg_is_hybrid_wanted(void) {
         return (wanted = r > 0 ? !b : is_default);
 }
 
+bool cg_legacy_force_enabled(void) {
+        bool force;
+
+        assert(cg_is_legacy_wanted());
```

Comment:
why assert on this? i'd make the function generic and return false if this test fails?